script.js

```diff
@@ -0,0 +1,428 @@
+/*Craig Bergmeyer
+Script.js
+INFO 2134
+Thoendel
+March 20, 2020*/
+
+
+window.addEventListener('load', function() {
+       
+    
+	//set function to handle a change in selection
+	let mealSelector = document.getElementById('meal');
+    
+    mealSelector.addEventListener('change', function(){
+    	// find the value of the selected option
+    	let mealSelected = mealSelector.options[mealSelector.selectedIndex].value;
+		let makeVis = document.getElementById('mealOptionsHolder');
+    	// make options visible based off of selected value
+    	if (mealSelected != ''){
+    		
+    		let makeVisBreak = document.getElementById('breakfastOptions');
+    		let makeVisLunch = document.getElementById('lunchOptions');
+    		let makeVisDinner = document.getElementById('dinnerOptions');
+    		makeVis.className = 'visible';
+    		// would probably also make some code to remove the radio button selected if meal is changed
+    		
+    		if (mealSelected == 'breakfast') {
+    			makeVisBreak.className = 'visible';
+    			makeVisLunch.className = 'hidden';
+    			makeVisDinner.className = 'hidden';    			
+    		}
+    		else if (mealSelected == 'lunch') {
+    			makeVisBreak.className = 'hidden';
+    			makeVisLunch.className = 'visible';
+    			makeVisDinner.className = 'hidden';	
+    		}
+    		else if (mealSelected == 'dinner') {
+    			makeVisBreak.className = 'hidden';
+    			makeVisLunch.className = 'hidden';
+    			makeVisDinner.className = 'visible';
+    		}
+    		
+    		
+    	}
+    	else {
+    			makeVis.className = 'hidden';
+    			
+    	}
```

Comment:
This works! Just something to think about...in this case we can overwrite the full className because we are really only using one class on these inputs at a time - but what if we weren't? What if we needed to apply something like a jQuery mobile class AND visible/hidden? 
Nice job ⭐️